doc/index.rst

```diff
@@ -413,6 +413,29 @@ References:
 - https://docs.gitlab.com/ce/ssh/README.html
 - https://docs.gitlab.com/ee/ci/ssh_keys/README.html
 
+(Github CI) Access to private repositories
```

Comment:
Nit; My personal style preference. Since there's another section for Gitlab CI for the same topic, can we make a section for private repo access, then move both Github and Gitlab under there?

Reply:
ok, will do

Reply:
done